src/main/java/org/broadinstitute/hellbender/engine/VariantWalker.java

```diff
@@ -65,8 +65,13 @@ private void initializeDrivingVariants() {
         } else {
             throw new UserException("File " + drivingVariantFile + " cannot be decoded as a variant file.");
         }
+    }
 
-        if ( hasIntervals() ) {
+    @Override
+    void initializeIntervals() {
+        super.initializeIntervals();
+        if (hasIntervals()) {
+            // propagate any intervals for traversal; this needs to happen AFTER the intervals have been set up
```

Comment:
Remove this fix the next time you rebase this branch -- a more general fix is about to be merged in https://github.com/broadinstitute/gatk/pull/976